core/io/image_loader.h

```diff
@@ -72,7 +72,7 @@ class ImageLoader {
 
 class ResourceFormatLoaderImage : public ResourceFormatLoader {
 public:
-	virtual RES load(const String &p_path, const String &p_original_path = "", Error *r_error = nullptr);
+	virtual RES load(const String &p_path, const String &p_original_path = "", Error *r_error = nullptr, bool p_no_subresource_cache = false);
```

Comment:
I'm not familiar with how the coding standards would decide on this (or *who* could confirm for you), but I'd want to double-check whether output variables should always be at the end of method definitions (in which case, `p_no_subresource_cache` would be preferred to be before `r_error`). On the other hand, doing so would require you to also refactor all uses of this loader in the codebase to ensure that it uses the proper ordering of arguments, so it'd be a lot more involved.

Reply:
Since this is 3.x I think it needs to be at the end, otherwise it'd be a compatibility breaking change.

Reply:
Compatibility is the main reason this was made into an optional parameter, yeah.

Reply:
It's not compatible if addon plugins have to include a new parameter (it's optional on call, not on override). It makes them break compat with older versions of Godot.